docs/sources/enterprise/aws-secrets-plugin.md

```diff
@@ -0,0 +1,87 @@
+---
+aliases:
+  - /docs/grafana/latest/enterprise/aws-secrets-plugin/
```

Comment:
Plugins that require GE and involve secret management have been moved from `/enterprise/` to sections under `/setup-grafana/`. This doc should be moved to the directory `docs/sources/setup-grafana/configure-security/configure-database-encryption/`.

To align with the other secret management docs in that directory, the filename should probably also be renamed to `encrypt-secrets-using-aws-secrets-manager.md`.

CC @chri2547 for additional feedback.

```suggestion
  - /docs/grafana/latest/setup-grafana/configure-security/configure-database-encryption/encrypt-secrets-using-aws-secrets-manager/
```

Reply:
moved file and updated alias